com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/IDebugSession.java

```diff
@@ -32,6 +32,8 @@
 
     IBreakpoint createBreakpoint(String className, int lineNumber, int hitCount);
 
+    IBreakpoint createBreakpoint(String className, int lineNumber, int hitCount, String condition);
```

Comment:
For the interface, just keep the interface concise. If I understand correctly, only this new one is enough?

Reply:
I will remove the other two unused apis.